pkg/azureconstants/azure_constants.go

```diff
@@ -103,10 +104,12 @@ const (
 	ControllerServiceAccountFinalizer = "disk.csi.azure.com/azuredisk-controller"
 	ReleaseNamespace                  = "kube-system"
 
-	CRIUpdateRetryDuration = time.Duration(1) * time.Second
-	CRIUpdateRetryFactor   = 3.0
-	CRIUpdateRetryStep     = 5
-	ZonedField             = "zoned"
+	CRIUpdateRetryDuration  = time.Duration(1) * time.Second
+	CRIUpdateRetryFactor    = 3.0
+	CRIUpdateRetryStep      = 5
+	ZonedField              = "zoned"
+	CRIDefaultMaxNetRetry   = 0
+	CRITransientMaxNetRetry = 5
```

Comment:
Consider `normalUpdateMaxNetRetry` and `forcedUpdateMaxNetRetry` to more closely align with the scenarios in which they're used. Also, do they need to be package public?

Reply:
Ah, I see that these are used by the `CrdProvisioner` and need to be public.